app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'AuthorId'
```

Comment:
- [ ] [optional] You don't have to add a foreign key here as it's defined in the schema.